### ISSUE TITLE
twisted.python.filepath.AbstractFilePath.getContent() docstring should be corrected

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8637]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8637|
|Type|enhancement|
|Created|2016-07-20 12:32:02Z|
|Branch|https://github.com/twisted/twisted/tree/8637-rodrigc-filepath-doc|

The docstring for twisted.python.filepath.AbstractFilePath.getContent()
says that getContent() retrieves a file-like object.  This is wrong,
it retrieves the contents of the file

<details><summary>Searchable metadata</summary>

```
trac-id__8637 8637
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__None None
branch__8637_rodrigc_filepath_doc 8637-rodrigc-filepath-doc
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1469017922221181 1469017922221181
changetime__1469076812152189 1469076812152189
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/365

**Comment by twisted-trac** [comment]
|[<img alt="markrwilliams's avatar" src="https://avatars.githubusercontent.com/u/704247?s=50" width="50" height="50">](https://github.com/markrwilliams)<a name="note_3"></a>|@markrwilliams commented|
|-|-|

Nice catch! The epytext and topfile both look good, and I didn't see any relevant failures in the documentation build. Please merge it!

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 66688e759ef0dfef415985c67c565485132ac066
```
#!CommitTicketReference repository="" revision="66688e759ef0dfef415985c67c565485132ac066"
Merge pull request #365 from twisted/8637-rodrigc-filepath-doc

Author: rodrigc
Reviewer: markrwilliams
Fixes: #8637

Clarify that AbstractFilePath.getContent() returns bytes
```
